src/index.scss

```diff
@@ -54,3 +54,21 @@ code {
 .text-danger {
   color: var(--color-brick-dark);
 }
+
+.button-icon {
+  background: none;
+  border: none;
+  color: inherit;
+  cursor: pointer;
+  font: inherit;
+  padding: var(--spacing-xs) var(--spacing-s);
+
+  &:hover {
+    background-color: var(--opening-period-listing-button-hover-background-color);
```

Comment:
Maybe we need to rename this color as it's now grown as a more generic color applied in many components?

Reply:
Added a specific variable